java/integTest/src/test/java/glide/cluster/CommandTests.java

```diff
@@ -788,10 +790,24 @@ public static Stream<Arguments> callCrossSlotCommandsWhichShouldFail() {
                         "fcall",
                         "7.0.0",
                         clusterClient.fcall("func", new String[] {"abc", "zxy", "lkn"}, new String[0])),
+                Arguments.of(
+                        "fcall",
```

Comment:
This this is just for logging purposes
```suggestion
                        "fcall binary",
```